### ISSUE TITLE
Add Genre specific parameter on the query

### ISSUE THREAD

**Issue by georgezalokostas** [omdbapi/OMDb-API#310]
Hello, I would like to contribute to this project and add such a functionality. Ping me If you're interested!

**Comment by MikeyA-yo** [comment]
i agree to this, if we could see a list of movies according to a particular genre
